packages/cli/test/unit/commands/alias.test.ts

```diff
@@ -0,0 +1,40 @@
+import { client } from '../../mocks/client';
+import alias from '../../../src/commands/alias';
+import { useUser } from '../../mocks/user';
+import { useAlias } from '../../mocks/alias';
+
+describe('alias', () => {
+  it('should list up to 20 aliases by default', async () => {
+    useUser();
+    useAlias();
+    client.setArgv('alias', 'ls');
+    const exitCodePromise = alias(client);
+    await expect(exitCodePromise).resolves.toEqual(0);
+    await expect(client.stderr).toOutput('dummy-19.app');
+  });
+
+  it('should list up to 2 aliases', async () => {
+    useUser();
+    useAlias();
+    client.setArgv('alias', 'ls', '--limit', '2');
+    const exitCodePromise = alias(client);
+    await expect(exitCodePromise).resolves.toEqual(0);
+    await expect(client.stderr).toOutput('dummy-1.app');
+  });
+
+  it('should throw an error if limit not valid', async () => {
+    useUser();
+    for (let limit of ['abc', '101']) {
+      client.setArgv('alias', 'ls', '--limit', limit);
+      const exitCodePromise = alias(client);
+      await expect(exitCodePromise).resolves.toEqual(1);
+    }
```

Comment:
**suggestion (non-blocking):** split this into two tests

I'm generally not a fan of generating tests at runtime. It makes them easier to write, sometimes harder to read, and often harder to debug.